data/lambeboluwatife.json

```
@@ -0,0 +1,74 @@
+{
+  "name": "Lambe Boluwatife",
+  "bio": "Frontend Developer | **Game** Enthusiast | **Bubble** Enthusiast | ReactJS | NextJS | NodeJS | MongoDB | Javascript | ExpressJS",
+  "tags": [
+    "Open Source",
+    "Javascript",
+    "Typescript",
+    "NextJS",
+    "ReactJS",
+    "NodeJS",
+    "MongoDB",
+    "ExpressJS"
+  ],
+  "socials": [
+    { "icon": "FaXTwitter", "url": "https://twitter.com/Danibholie" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/boluwatife-lambe-87b0b6136/" },
+    { "icon": "FaGithub", "url": "https://github.com/lambeboluwatife" },
+    { "icon": "FaYoutube", "url": "https://www.youtube.com/@LBDmedia" }
+  ],
+  "links": [
+    {
+      "name": "View my Portfolio",
+      "url": "https://boluwatifelambe.netlify.app/",
+      "icon": "FaLink"
+    },
+    {
+      "name": "Follow my Open Source on GitHub",
+      "url": "https://github.com/lambeboluwatife",
+      "icon": "FaGithub",
+      "color": "#171515"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Danibholie",
+      "icon": "FaXTwitter",
+      "color": "#00ACEE"
+    },
+    {
+      "name": "Learn more about development on my YouTube channel",
+      "url": "https://youtube.com/@LBDmedia",
+      "icon": "FaYoutube",
+      "color": "#FF0000"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://linkedin.com/in/boluwatife-lambe-87b0b6136/",
+      "icon": "FaLinkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Software Development",
+      "date": "Dec 2023",
+      "icon": "FaCode",
+      "description": "Completed Software Development Program",
+      "url": "https://boluwatifelambe.netlify.app/"
+    },
+    {
+      "title": "Started YouTube",
+      "image": "https://github.com/lambeboluwatife.png",
+      "date": "Dec 2022",
+      "icon": "FaYoutube",
+      "description": "First real video",
+      "url": "https://youtube.com/@LBDmedia"
+    },
+    {
+      "title": "Open Source",
+      "date": "2023",
+      "icon": "FaGithub",
+      "description": "First Open Source Contribution",
+      "url": "https://discord.gg/4c-784142072763383858"
+    },
```

```suggestion
    }
```
The trailing comma isnot required